LayoutTests/resources/testharnessreport.js

```diff
@@ -14,7 +14,7 @@
 
 // Setup for WebKit JavaScript tests
 if (self.testRunner) {
-    testRunner.dumpAsText();
+    testRunner.dumpAsText(testRunner.shouldDumpPixels());
```

Comment:
Just for my knowledge asking - LayoutTest use this but WPT in import use `LayoutTests/imported/w3c/web-platform-tests/resources/testharnessreport.js`?

Reply:
No, the WPT tests use this too when run under `run-webkit-tests`, it's how WPT tests know how to interact with `WebKitTestRunner`.